xnmt/specialized_encoders/segmenting_encoder/segmenting_encoder.py

```diff
@@ -102,7 +102,14 @@ def transduce(self, embed_sent: ExpressionSequence) -> List[ExpressionSequence]:
         sent_context = self.final_transducer.transduce(expr_seq)
         self.final_states.append(self.final_transducer.get_final_states())
         enc_outputs.append(sent_context)
-      return CompoundSeqExpression(enc_outputs)
+      # TODO (neubig): This is a temporary minimial fix to revert to using only one sample.
+      #                Multiple samples can be done by the MLELoss(repeat=samp_num) functionality.
```

Comment:
I can fix this if you have decided about the losses from multiple samples. So it means that nothing should really be changed in the calc_additional_loss part because it still needs a single loss from each of the sample. And the same as in policy_gradient.py

Reply:
Yeah, it'd be great if you could fix this later when I'm done with my changes to the interface. I just wanted to check with you first to make sure I didn't do anything that would make it difficult for you to do what you want to do.